tests/e2e_tests/test_ui.py

```diff
@@ -342,10 +354,11 @@ def get_args():
     test.create_artifacts()
     test.test_login_create_user()
     test.test_click_header_buttons()
-    test.test_artifact_upload()
-    test.test_authorize_all()
-    test.test_basic_inventory()
-    test.test_deploy()
-    test.test_deployment_in_progress()
-    test.test_deployment_successful()
-    #pytest.main(args=[os.path.realpath(__file__)])#, "--url", args.url])
+    # TODO adjust to match correct ui elements
+    # test.test_artifact_upload()
+    # test.test_authorize_all()
+    # TODO adjust to match correct ui elements
+    # test.test_basic_inventory()
+    # test.test_deploy()
+    # test.test_deployment_in_progress()
+    # test.test_deployment_successful()
```

Comment:
Apart from removing these from here, we should mark them as skip for pytest

So for each of these tests, please add `@pytest.mark.skip` in the method declarations. You will also need to import pytest

Reply:
As discussed in person, we cannot run them using pytest due to the dependencies. But can we add the decorators anyway?